### DISCUSSION TITLE
Multiple connectable instances

### DISCUSSION THREAD

**Discussion by teceP** [seaweedfs/seaweedfs#3401] [Q&A]
Is it possible to have multiple instances, which can communicate with each other?
Instance A runs in a DMZ internal zone and would keep sensible data.
Instance B runs external and keeps everything else.
Sometimes instance A would send sensible data to instance B. This data must be deleted after a certain time (TTL?).

**Comment by chrislusf** [comment]
Your requirement is too specific to satisfy.
